docs/source/trainer/using_the_trainer.rst

```diff
@@ -443,6 +443,45 @@ points during training and (2) load them back to resume training later.
 This was just a quick tour of all the features within our trainer. Please see the other
 guides and notebooks for more information.
 
+Reproducibility
+~~~~~~~~~~~~~~~
+
+The random seed can be provided to the trainer directly, e.g.
+
+.. code:: python
+
+    from composer import Trainer
+
+    trainer = Trainer(
+        ...,
+        seed=42,
+    )
+
+If no seed is provided, a random seed will be generated from system time.
+
+Since the model and dataloaders are initialized outside of the Trainer, for complete
+determinism, we recommend calling :meth:`~.reproducibility.seed_all` and/or
```

Comment:
These are not autolinking...see https://mosaicml-composer--794.com.readthedocs.build/en/794/trainer/using_the_trainer.html#reproducibility